test_local_deployment.sh

```diff
@@ -0,0 +1,10 @@
+set -ex
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6003355)